client/src/components/ImageUpload/ImageUpload.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// @ts-nocheck
+import Dropzone, { DropzoneState } from 'react-dropzone';
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import { useState } from 'react';
+
+interface Props {
+  handleUpload: (files: File[]) => void;
+}
+
+//lift up the state
+
+const ImageUpload = ({ handleUpload }: Props): JSX.Element => {
+  const classes = useStyles();
+  const [files, setFiles] = useState([]);
+
+  const onDrop = (acceptedFiles) => {
+    setFiles(
+      acceptedFiles.map((file) =>
+        Object.assign(file, {
+          preview: URL.createObjectURL(file),
+        }),
+      ),
+    );
+  };
+
+  const previews = () => {
+    return files.map((file: File) => (
+      <div className={classes.thumb} key={file.name}>
+        <div className={classes.thumbInner}>
+          <img src={file.preview} className={classes.img} />
+        </div>
+      </div>
```

Comment:
Please use MUI `Box` component instead of this HTML `div`

Reply:
Please address this 😄